docs/client/installation/windows-installation.md

```diff
@@ -3,8 +3,8 @@
 
 ### With 0install (recommended way)
 1. Click [here](http://0install.de/files/zero-install.exe) to download and install [0Install for Windows](http://0install.net/install-windows.html).
-2. Run `0install https://raw.githubusercontent.com/zanata/zanata.github.io/master/files/0install/zanata-cli-jre.xml` to setup alias.
-3. Now you can run `zanata-cli.bat --help` for more options.
+2. Run `0install add zanata-cli https://raw.githubusercontent.com/zanata/zanata.github.io/master/files/0install/zanata-cli-jre.xml` to set up an alias `zanata-cli`.
```

Comment:
The alias feature does not appear to be part of the add command in the Windows port of 0install